src/Workspaces/Core/Portable/FindSymbols/SymbolFinder_Declarations.cs

```diff
@@ -52,11 +52,11 @@ public static partial class SymbolFinder
                 return ImmutableArray<ISymbol>.Empty;
             }
 
-            return await FindDeclarationsAsync(
+            return await FindAllDeclarationsWithNormalQueryAsync(
                 project, SearchQuery.Create(name, ignoreCase), filter, cancellationToken: cancellationToken).ConfigureAwait(false);
         }
 
-        internal static Task<ImmutableArray<ISymbol>> FindDeclarationsAsync(
+        internal static Task<ImmutableArray<ISymbol>> FindAllDeclarationsWithNormalQueryAsync(
```

Comment:
Nit: `Standard` might be better than `Normal`.  Also a doc comment explaining what a `Normal` query is might be helpful for someone first approaching the code.